### DISCUSSION TITLE
Line by line comparison

### DISCUSSION THREAD

**Discussion by pauvt** [WinMerge/winmerge#512] [Q&A]
It be very useful to be to force winmerge to compare a text file contents line by line, rather than it try and auto align the differences.

In the following example I want to auto-merge the three files but winmerge can't because in some of the files I've added a new line.

![image](https://user-images.githubusercontent.com/58662223/102367577-388a8080-3fba-11eb-971b-4297e4e015c1.png)

Am I missing something or is this a bug/improvement?

**Comment by sdottaka** [comment]
It's not a bug, but it needs improvement.

You can get the same result with the diff3 command.
kdiff3 (http://kdiff3.sourceforge.net/) seems to do this well.

**Reply by zengfanfan** [reply]
want this too.

**Comment by LAPIII** [comment]
I created a [discussion ](https://github.com/WinMerge/winmerge/discussions/1245 ) about this in Ideas. Please upvote.
